web/client/themes/default/less/geostory.less

```diff
@@ -296,6 +299,7 @@
     }
     .ms-section.ms-expandable-background {
         &.ms-section-title .ms-content-text,
+        &.ms-section-banner .ms-content-text,
```

Comment:
I think this style is not needed because it applies to content style` .ms-content-text`